support-frontend/assets/pages/showcase/components/ctaSubscribe.jsx

```diff
@@ -22,7 +23,14 @@ export default function CtaSubscribe() {
         </p>
       </Text>
       <NarrowContent>
-        <AnchorButton aria-label={null} icon={<ArrowRightStraight />} href="/subscribe">Choose a Subscription</AnchorButton>
+        <AnchorButton
+          aria-label={null}
+          icon={<ArrowRightStraight />}
+          href="/subscribe"
+          onClick={() => sendClickedEvent('support_page_cta_subscribe')()}
```

Comment:
There is also a function called `sendTrackingEventsOnClick` in subscriptions.js which does a very similar thing but also sends data to Ophan. Not sure which is the best bet in this case, probably the one you've picked?

Reply:
Yes - I saw that function when I was reviewing your PR! I think it's less suitable as it wants you to specify a product as opposed to clicking on 'subscribe' or 'contribute'